src/libraries/System.Private.CoreLib/src/System/Guid.cs

```diff
@@ -243,7 +243,7 @@ internal readonly void SetFailure(ParseFailure failureKind)
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             public readonly Guid ToGuid()
             {
-                return Unsafe.As<GuidResult, Guid>(ref Unsafe.AsRef(in this));
+                return Unsafe.ReadUnaligned<Guid>(ref Unsafe.As<GuidResult, byte>(ref Unsafe.AsRef(in this)));
```

Comment:
this could be Unsafe.Bitcast I guess (casting GuidResult to Guid)

Reply:
`GuidResult` has a larger size.

https://github.com/dotnet/runtime/blob/7f000274572f96eea8083ae723dd81f634443fc4/src/libraries/System.Private.CoreLib/src/System/Guid.cs#L209

Reply:
ah, indeed